lib/utils/logDeprecation.js

```diff
@@ -2,8 +2,12 @@
 
 const chalk = require('chalk');
 const weakMemoizee = require('memoizee/weak');
+const _ = require('lodash');
+const ServerlessError = require('../serverless-error');
 
-const disabledCodesByEnv = extractCodes(process.env.SLS_DEPRECATION_DISABLE);
+const disabledDeprecationCodesByEnv = extractCodes(process.env.SLS_DEPRECATION_DISABLE);
+
+const notificationModeByEnv = process.env.SLS_DEPRECATION_NOTIFICATION_MODE;
```

Comment:
Should we also validate if provided mode via ENV VAR is correct (one of `warn`, `error`)?

Reply:
I think convention is to not validate env vars very strictly (do we validate them anywhere?). If value doesn't expose recognized format we rather ignore it, and there's no even warning.

I guess it's a convention because env vars, unlike CLI options, might be set by some side entities

Reply:
I don't think we do validate them anywhere, I was asking mostly from curiosity if we should even consider such validation in general.